src/layers/conv.jl

```diff
@@ -393,3 +393,43 @@ function Base.show(io::IO, l::EdgeConv)
     print(io, ", aggr=", l.aggr)
     print(io, ")")
 end
+
+
+"""
+    GINConv([graph, ] nn, eps, train_eps
+"""
+
+struct GINConv{V<:AbstractFeaturedGraph,R<:Real} <: MessagePassing
+    fg::V
+    nn
+    eps::Untrainable{R}
+end
```

Comment:
there is no need to introduce a new `Untrainable` type, since Flux won't consider scalar numbers as trainable parameters.
There are some examples of this in Flux, e.g. see the normalization layers 
https://github.com/FluxML/Flux.jl/blob/master/src/layers/normalise.jl

Reply:
Removed, instead changed `Flux.trainable(g::GINConv)`